kimchi/src/circuits/wires.rs

```diff
@@ -6,13 +6,13 @@ use std::array;
 use std::io::{Read, Result as IoResult, Write};
 
 /// Number of registers
-pub const COLUMNS: usize = 15;
+pub const KIMCHI_COLS: usize = 15;
```

Comment:
I'm not really a fan of this name. Kimchi is the proof system, so any number of columns that you choose could justifiably be called `KIMCHI_COLS`. Prefer `BERKELEY_COLUMNS` perhaps?